src/components/Header/HeaderContact.js

```diff
@@ -90,12 +90,16 @@ const styles = StyleSheet.create({
   }),
   text: {
     color: COLORS.black,
-    fontFamily: fonts.inter[600],
+    fontFamily: fonts.inter[500],
     fontSize: normalizeFontSize(14),
-    fontWeight: 'bold',
     padding: 10,
     textAlign: 'center'
   },
+  titleText: {
+    position: 'absolute',
+    width: '100%',
+    zIndex: -1
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here seem to adjust the font family from `fonts.inter[600]` to `fonts.inter[500]`, remove the bold weight, and add a new style `titleText`. These changes are likely intended to adjust the visual appearance of the text. However, ensure that these changes align with your design system or style guide.
```diff
- fontFamily: fonts.inter[600],
- fontWeight: 'bold',
+ fontFamily: fonts.inter[500],

+ titleText: {
+   position: 'absolute',
+   width: '100%',
+   zIndex: -1
+ },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->